.changeset/sweet-lamps-cover.md

```diff
@@ -0,0 +1,12 @@
+---
+'@astrojs/tailwind': minor
+'@astrojs/markdoc': minor
+'@astrojs/netlify': minor
+'@astrojs/svelte': minor
+'@astrojs/image': minor
+'@astrojs/mdx': minor
+'@astrojs/vue': minor
+'astro': minor
```

Comment:
These should all be patches because we are not adding new features.

Reply:
This has been updated, and all tests are now green.

Reply:
@ematipico , is there anything else that should be changed here?